src/aws-cpp-sdk-core/include/smithy/identity/resolver/built-in/DefaultAwsCredentialIdentityResolver.h

```diff
@@ -9,33 +9,51 @@
 #include <aws/core/auth/AWSCredentials.h>
 #include <aws/core/auth/AWSCredentialsProviderChain.h>
 
+namespace{
+    constexpr char ALLOC_ID[] = "DefaultAwsCredentialIdentityResolver";
```

Comment:
why is this in the anonymous namespace and not in the smithy namespace?

Reply:
this is in a .h file, and I think there are other strings with same name, this tag is used only internal to that resolver

Reply:
> I think there are other strings with same name
 
 then lets name it something else instead of creating a anon namespace